backend/src/main/resources/graphql/main.graphqls

```diff
@@ -344,6 +346,7 @@ type Mutation {
     county: String
     race: String @Size(min: 0, max: 32)
     ethnicity: String @Size(min: 0, max: 32)
+    tribalAffiliation: String @Size(min: 0, max: 6)
```

Comment:
It was more or less arbitrary! I'll set it to `3`, though, since the max value is `566`. I just didn't feel right leaving it at `32`.

Note to self that this may change depending on what decision we make about the data model...